template.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
<span lang="fr">Est ce que tu peux vérifier pour intégrer ce template à même le Gruntfile.coffee file?</span>

Reply:
Quel serait l'avantage ?

Reply:
Ça éviterai de la confusion et du questionnement future par rapport à avoir un fichier nommée "template.json" sur le root. Il n'est pas claire, avec le nom de fichier "template.json" que ce fichier est seulement utilisé par le build script. Aussi, en l'intégrant au Gruntfile.coffee, l'ensemble des configurations pour le build script vont être dans un document central. 

Un document coffee est aussi un document javacript. Donc tu devrais être capable d'y intégrer ce template dans une variable local

Reply:
@shawnthompson @duboisp 

I made the script relative.

Do you want to add a JSON file only to themes-cdn ?

	@registerTask(
		"add-json"
		"add json file"
		->
		[
			pkgOriginal = grunt.file.readJSON("package.json");
			addTo = "themes-cdn";
			obj = {
				name: "theme-" + pkgOriginal.name.toLowerCase(),
				version: pkgOriginal.version,
				description: pkgOriginal.name.toLowerCase() + " theme"
				repository: {
					type: "git",
					url: "git+https://github.com/wet-boew/" + addTo + ".git"
				},
				author: "wet-boew-bot",
				license: "MIT",
				bugs: {
					url: "https://github.com/wet-boew/" + pkgOriginal.name.toLowerCase() + "/issues"
				},
				homepage: "https://github.com/wet-boew/" + addTo + "#readme"
			};			
			grunt.file.write("dist/GCWeb/package.json", JSON.stringify(obj, null, 2));
		]
	);

Result:

{
  "name": "theme-gcweb",
  "version": "5.1.0-development",
  "description": "gcweb theme",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/wet-boew/themes-cdn.git"
  },
  "author": "wet-boew-bot",
  "license": "MIT",
  "bugs": {
    "url": "https://github.com/wet-boew/gcweb/issues"
  },
  "homepage": "https://github.com/wet-boew/themes-cdn#readme"
}

Reply:
And does the name is ok  "theme-gcweb", or it will confict with another one ?

https://github.com/wet-boew/theme-gcweb

Reply:
What about a name like: "wet-boew-theme-canada.ca" ?